src/WebJobs.Script.WebHost/Properties/Resources.resx

```diff
@@ -183,7 +183,7 @@
   </data>
   <assembly alias="System.Windows.Forms" name="System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
   <data name="Homepage" type="System.Resources.ResXFileRef, System.Windows.Forms">
-    <value>..\Home.html;System.String, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089;utf-8</value>
+    <value>../Home.html;System.String, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089;utf-8</value>
```

Comment:
Yes. The Mono resx compiler doesn't transparently remap backslashes, and IIRC the Windows one can deal with forward slashes.